framework/include/cppmicroservices/Framework.h

```diff
@@ -140,7 +148,9 @@ class US_Framework_EXPORT Framework : public Bundle
      *
      * @throws std::runtime_error If this Framework could not be started.
      */
+#ifdef DOXYGEN_RUN
```

Comment:
There is no particular reason... making the ifdef scope as small as possible felt right to me.